android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -169,7 +174,14 @@ public void onError(int i) {
         );
 
         if (cameraCapturer.getSupportedFormats().size() > 0) {
-            localVideoTrack = LocalVideoTrack.create(getContext(), true, cameraCapturer);
+         VideoConstraints videoConstraints = new VideoConstraints.Builder()
```

Comment:
Is this correctly aligned in the if block ? Maybe it's github not rendering correctly.